docker/README.md

```diff
@@ -48,7 +66,7 @@ And initialize the metadata store that bookies will use to store information:
 docker run -it --rm \
     --network "my-bookkeeper-network" \
     --env ZK_URL=my-zookeeper:2181 \
-    bookkeeper \
+    apache/bookkeeper:4.5.0 \
     bookkeeper shell metaformat
```

Comment:
Thanks for change this @ArvinDevel .
line 64--71 is not necessary for the new image. We could remove it, or change it to something like line-17: 
```
-c "/opt/bookkeeper/bin/bookkeeper shell metaformat -n -f"
```